### ISSUE TITLE
Quickwit 0.3 not working on Minio 

### ISSUE THREAD

**Issue by fulmicoton** [quickwit-oss/quickwit#1595]
As reported by @kstaken on discord, quickwit 0.3 does not work with minio through the custom endpoint trick.
(not reproduced yet)

**Comment by kstaken** [comment]
I also dumped the put request and I noticed that the content-length here doesn't match the size logged for the split. It's not clear to me if this is relevant but the content-length is 17349530 bytes while the size of the split in the metastore is 26926583. Metastore does show the footer_offsets.end as 17349530. Shouldn't the end of the footer correspond to the size of the split?

```
request PutObjectRequest { acl: None, body: Some(<ByteStream size_hint=None>), bucket: "qwdata", bucket_key_enabled: None, cache_control: None, content_disposition: None, content_encoding: None, content_language: None, content_length: Some(17349530), content_md5: None, content_type: None, expected_bucket_owner: None, expires: None, grant_full_control: None, grant_read: None, grant_read_acp: None, grant_write_acp: None, key: "indexes/data1/01G4JJGCGSRJVB306TTYTTBNYE.split", metadata: None, object_lock_legal_hold_status: None, object_lock_mode: None, object_lock_retain_until_date: None, request_payer: None, sse_customer_algorithm: None, sse_customer_key: None, sse_customer_key_md5: None, ssekms_encryption_context: None, ssekms_key_id: None, server_side_encryption: None, storage_class: None, tagging: None, website_redirect_location: None }
```

```
 01G4JJGCGSRJVB306TTYTTBNYE | Staged      |                  |                | {}   | {"version":"1","split_id":"01G4JJGCGSRJVB306TTYTTBNYE","num_docs":100000,"size_in_bytes":26926583,"time_range":null,"create_timestamp":1654185862,"tags":[],"demux_num_ops":0,"footer_offsets":{"start":17320770,"end":17349530}} | data1    | 2022-06-02 16:04:22.218894 | 2022-06-02 16:04:22.218894
```

**Comment by kstaken** [comment]
Finally figured out what this is. It's caused by the trailing slash on the URL in `QW_S3_ENDPOINT`.

```
export QW_S3_ENDPOINT=http://192.168.86.220:9000/
```

**Comment by fulmicoton** [comment]
@kstaken Thanks for the report and sorry for the trouble. The fix will be shipped in 0.3.1
